app/controllers/books_controller.rb

```diff
@@ -4,6 +4,8 @@ def index
   end
 
   def show
-    @book = Book.find(params[:id])
+    @book = Book
+      .includes(reviews: :user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.